setup.cfg

```diff
@@ -37,6 +37,25 @@ install_requires =
     rp2>=0.9.25
     yfinance>=0.1.70
 
+[options.extras_require]
+dev =
```

Comment:
managing dev dependency using extras is more idiomatic across python projects. we also don't need to repeat the install_requires dependencies in the requirements.txt file. In fact, I think we can just get rid of the requirements.txt file and use `pip install '.[dev]'` everywhere :)

Reply:
Didn't know this, sounds good. Could you update also the Makefile (which is using requirements.txt)?